packages/react-inbox/src/components/Messages/styled.ts

```diff
@@ -46,5 +46,41 @@ export const Empty = styled.div`
   text-align: center;
   color: #73819B;
   margin: auto;
-  height: 100%;
 `;
+
+
+export const Footer = styled.div(({ theme }) => ({
+  "borderBottomLeftRadius": 24,
+  "borderBottomRightRadius": 24,
+  "height": 45,
+  "fontSize": "10px",
+  "fontStyle": "normal",
+  "fontWeight": "700",
+  "lineHeight": "14px",
+  "letterSpacing": "0em",
+  "textAlign": "left",
+  "color": "#B9C0CD",
+  "display": "flex",
+  "alignItems": "flex-end",
+  "justifyContent": "flex-end",
+  "> div": {
```

Comment:
could we perhaps add a more explicit selector here to indicate what this div stands for?

Reply:
Fixed.